### ISSUE TITLE
Carto branding is everywhere

### ISSUE THREAD

**Issue by almereyda** [kidwellj/mapping.community-carto_install_doc#26]
When displaying a published map,

- [ ] the bottom shows a Carto logo pointing at their site, and
- [ ] the attribution reads *© Carto*.

This is loosely related to #24.

**Comment by kidwellj** [comment]
That is a reference to the copyright on basemap tiles, I believe. Can you try using a different basemap and see if the attribution disappears? To be fair to Carto, if they've developed the basemap, it's only fair to provide attribution...

**Comment by almereyda** [comment]
No, this is also shown for Stamen Toner backgrounds, and attributes those correctly.

- https://carto.mapping.community/user/yala/builder/a7418575-ed20-4568-9a4b-e31f7a172c2f/embed

**Comment by kidwellj** [comment]
Thanks for checking - I think I've found the issue (here: https://github.com/splashblot/dronedb/issues/68). Can you check and see if it's resolved now?

**Comment by kidwellj** [comment]
For reference, fix I've applied was:

`RAILS_ENV=production bundle exec rake cartodb:features:add_feature_flag[disabled_cartodb_logo]`

**Comment by almereyda** [comment]
The map that I linked above still looks the same. Maybe one needs to apply a patch to the code somehow?

**Comment by kidwellj** [comment]
@almereyda Yep, have already tried for whole organisation. Might be necessary to push the platform to a new version before the rake feature will work. Realistically, I'm looking at running the next upgrade in October '18 or so...

**Comment by kidwellj** [comment]
Opening new issue to track upgrade progress: #27
